python/ray/train/torch/torch_predictor.py

```diff
@@ -104,11 +104,25 @@ def _array_to_tensor(
     def _tensor_to_array(self, tensor: torch.Tensor) -> np.ndarray:
         return tensor.cpu().detach().numpy()
 
+    @DeveloperAPI
     def _model_predict(
```

Comment:
```suggestion
    def _call_model(
```

Would this be more clear?

Reply:
yep this sg, will make the change!

Reply:
+1, and shouldn't we remove the leading underscore if we're promoting this to a developer API, or is that not our standard?

Reply:
+1 to remove underscore.

Reply:
@clarkzinzow yeah we had this discussion before for `_predict_pandas`. The convention we've been following so far is to have DeveloperAPIs also have a leading underscore.

I would prefer removing the leading underscore for developer APIs, since they are explicitly annotated as Developer APIs anyways. But also cc @ericl for thoughts

Reply:
wait @ericl you previously pushed for Developer APIs having leading underscores for `_predict_pandas`?

I agree with making the change, but just want to clarify exactly what the convention we are going with.

Reply:
https://github.com/ray-project/ray/pull/25425#discussion_r888538869

Reply:
(and once we define a convention, to stick to that across all our APIs)

Reply:
Got it. Maybe we should just make this a Public API then, what do you think? Basically, end users may need to use this, but they never should need to touch the other predict methods.

Reply:
ok yeah makes sense.

Reply:
Yep that sounds best to me.

Reply:
updated